### PR TITLE
High-Resolution Radiosonde BUFR Tables

### PR DESCRIPTION
**Pull request by PraveenKumar-NOAA**
## Description

This PR provides BUFR tables for each high-resolution radiosonde data subset, which are described below: 

- NC002101 -- MSG TYPE 002-101 RAWINSONDE - FIXED LAND
- NC002102 -- MSG TYPE 002-102 RAWINSONDE - MOBIL LAND
- NC002103 -- A51103 | MSG TYPE 002-103 RAWINSONDE - SHIP
- NC002104 -- A51104 | MSG TYPE 002-104 DROPWINSONDE
- NC002105 -- A51105 | MSG TYPE 002-105 PIBAL

**The following files (BUFR Tables) are new:**

- adpupa.nc002101.bufr.table.txt
- adpupa.nc002102.bufr.table.txt
- adpupa.nc002103.bufr.table.txt
- adpupa.nc002104.bufr.table.txt
- adpupa.nc002105.bufr.table.txt

Information for the mnemonics, units, descriptions, etc. is provided. These tables are needed to write a YAML for the _bufr2ioda_ conversion. 

### Issue(s) addressed

#1113 

## Acceptance Criteria (Definition of Done)

It will be done once all-important mnemonics are determined and documented.

## Dependencies

There are no dependencies.

### CODE REVIEW

File: doc/bufr_table_samples/adpupa.nc002101.bufr.table.txt [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+|------------------------------------------------------------------------------|
+| MNEMONIC | NUMBER | DESCRIPTION                                              |
+|----------|--------|----------------------------------------------------------|
+|          |        |                                                          |
+| NC002101 | A51101 | MSG TYPE 002-101  RAWINSONDE - FIXED LAND (BUFR)         |
+|          |        |                                                          |
+| WMOBLKST | 301001 | IDENTIFICATION - WMO BLOCK AND STATION NUMBER            |
+| YYMMDD   | 301011 | DATE -- YEAR, MONTH, DAY                                 |
+| HHMMSS   | 301013 |                                                          |
+| LTLONH   | 301021 |                                                          |
```
Comment (nicholasesposito) [inline doc/bufr_table_samples/adpupa.nc002101.bufr.table.txt:12]: Is there no description for lines 11 and 12?

File: doc/bufr_table_samples/adpupa.nc002102.bufr.table.txt [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+| NC002102 | A51102 | MSG TYPE 002-102  RAWINSONDE - MOBIL LAND (BUFR)         |
+|          |        |                                                          |
+| WMOBLKST | 301001 | IDENTIFICATION - WMO BLOCK AND STATION NUMBER            |
+| YYMMDD   | 301011 | DATE -- YEAR, MONTH, DAY                                 |
+| HHMMSS   | 301013 |                                                          |
+| LTLONH   | 301021 |                                                          |
+| IDLSIPTM | 301111 | IDENTIFICATION OF LAUNCH SITE AND INSTRUMENTATION        |
+| DATETMLN | 301113 | DATE/TIME OF LAUNCH                                      |
+| HAVCOLS  | 301114 | HORIZONTAL AND VERTICAL COORDINATES OF LAUNCH SITE       |
+| INFORDAS | 301128 | Additional information on radiosonde ascent              |
```
Comment (nicholasesposito) [inline doc/bufr_table_samples/adpupa.nc002102.bufr.table.txt:16]: Some descriptions are capitalized and some are not. Is this ok?

**Review by nicholasesposito — COMMENTED** [review]
Just a few questions. If those are good, then I'll approve.

File: doc/bufr_table_samples/adpupa.nc002102.bufr.table.txt (hunk 2 above)
Comment (PraveenKumar-NOAA) [reply doc/bufr_table_samples/adpupa.nc002102.bufr.table.txt:16]: @nicholasesposito same as described above.

Comment (nicholasesposito) [reply doc/bufr_table_samples/adpupa.nc002102.bufr.table.txt:16]: Got it. I'll approve.

**Review by nicholasesposito — APPROVED** [review]

**Review by srherbener — APPROVED** [review]

**Review by emilyhcliu — APPROVED** [review]

**Review by BenjaminRuston — APPROVED** [review]

**Comment by PraveenKumar-NOAA** [comment]
@srherbener please merge this PR. This PR is only for documentation, i.e. BUFR tables.

**Pull request merged**